diesel/src/pg/backend.rs

```diff
@@ -78,7 +78,7 @@ impl PgTypeMetadata {
     ///
     /// [OID]: https://www.postgresql.org/docs/current/static/datatype-oid.html
     pub fn array_oid(self) -> Result<u32, impl std::error::Error + Send + Sync> {
```

Comment:
Seems i've missed that now twice. That should match the oid function above.

```suggestion
     pub fn array_oid(&self) -> Result<u32, impl std::error::Error + Send + Sync> {
```